platform/use-platform/virtual-clusters/add-host-cluster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Add existing host clusters
+sidebar_label: Add Existing Host Clusters
+sidebar_position: 3
+description: Add externally deployed host clusters to Platform with visibility-only or full management capabilities
+---
+
+import PageVariables from '@site/src/components/PageVariables';
+import InterpolatedCodeBlock from '@site/src/components/InterpolatedCodeBlock';
+
+You can register existing host clusters to an installed vCluster Platform running in another host cluster.  
```

Comment:
Start by explaining what a host cluster is. Also this should be more specific like `you can add an existing Kubernetes cluster as a deployment target for virtual clusters. In the platform those are called host clusters.

The part `vCluster Platform running in another host cluster`  is confusing, it is a given that platform deployment runs somewhere, so no need to specify this part.